calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbRules.java

```diff
@@ -688,8 +709,9 @@ private static Operand reformatObjectIdLiteral(
         }
 
         private static Operand translateCurrentTimestamp(final Instant currentTime) {
-            return new Operand("{\"$date\": {\"$numberLong\": "
-                    + "\"" + currentTime.toEpochMilli() + "\"}}");
+            final String currentTimestamp = "{\"$date\": {\"$numberLong\": "
```

Comment:
current_time, current_date, current_timestamp can be treated as literals and can be used on the right of query comparison operators